npm/react/cypress/component/basic/error-boundary.jsx

```diff
@@ -1,46 +1,46 @@
-import React from 'react'

```

Comment:
Change from running `git add --renormalize . ` after adding .gitattributes file to normalize line endings